lib/src/runner/configuration.dart

```diff
@@ -50,6 +50,9 @@ class Configuration {
   bool get pauseAfterLoad => _pauseAfterLoad ?? false;
   final bool _pauseAfterLoad;
 
+  /// The path to a file to load as a base configuration before this one.
+  final String includePath;
```

Comment:
Rather than adding a field that only gets set and used in `load.dart`, it's probably cleaner to just handle the `include` field specially in the `load()` function.

Reply:
I wasn't sure if its useful to have this just to be able to make configuration issues more discoverable (i.e. "this file originated from that file"). Happy to make that change though, so done.